apps/docs/content/frameworks/laravel/redis.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Using Redis with Laravel
+description: Learn how to configure and use Redis for caching, queues, and sessions in your Laravel application on Zerops
+---
+
+import Image from '/src/components/Image';
+
+# Redis Integration with Laravel
+
+Redis is a powerful in-memory data structure store that can be used as a database, cache, message broker, and queue. This guide will help you integrate Redis with your Laravel application on Zerops, enabling high-performance caching, session management, and queue processing.
+
+## Adding Redis Service
+
+Redis needs to be added as a service to your Zerops project before you can use it with Laravel. The service configuration defines the Redis version, availability mode, and other essential settings.
```

Comment:
I believe not everyone knows valkey, add a sentence why there valkey instead of proper redis